SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -11097,6 +11387,249 @@ void CEulerSolver::BC_ActDisk(CGeometry *geometry, CSolver **solver_container, C
 
 }
 
+void CEulerSolver::BC_ActDisk_VariableLoad(CGeometry *geometry, CSolver **solver_container, CNumerics *conv_numerics, CNumerics *visc_numerics,
+                              CConfig *config, unsigned short val_marker, bool val_inlet_surface) {
+
+  /*!
+   * \function BC_ActDisk_VariableLoad
+   * \brief Actuator disk model with variable load along disk radius.
+   * \author: E. Saetta, L. Russo, R. Tognaccini (GitHub references EttoreSaetta, lorenzorusso07, rtogna).
+   * Theoretical and Applied Aerodynamics Research Group (TAARG), University of Naples Federico II.
+   * \version 7.0.5 “Blackbird”
+   * First release date : July 1st 2020
+   * modified on:
+   *
+   * Force coefficients distribution given in an input file. Actuator disk data initialized in function SetActDisk_BCThrust.
+   * Entropy, acoustic Riemann invariant R+ and  tangential velocity extrapolated  from upstream flow;
+   * acoustic Riemann invariant R- is extrapolated from downstream.
+   * Hovering condition simulation not available yet: freestream velocity must be different than zero.
+   */
+
+  unsigned short iDim;
+  unsigned long iVertex, iPoint, GlobalIndex_donor, GlobalIndex;
+  su2double Pressure, Velocity[MAXNDIM],
+  Velocity2, Entropy, Density, Energy, Riemann, Vn, SoundSpeed, Vn_Inlet,
+  Area, UnitNormal[MAXNDIM] = {0.0}, *V_outlet, *V_domain, *V_inlet;
+
+  su2double Pressure_out, Density_out,
+  Pressure_in, Density_in;
+
+  su2double C[MAXNDIM], Prop_Axis[MAXNDIM], R, r[MAXNDIM], r_;
+  su2double Fa, Fx, Fy, Fz;
+  su2double u_in, v_in, w_in, u_out, v_out, w_out, uJ, vJ, wJ;
+  su2double Temperature_out, H_in, H_out;
+  su2double FQ, Q_out, Density_Disk;
+  su2double SoSextr, Vnextr[MAXNDIM], Vnextr_, RiemannExtr, QdMnorm[MAXNDIM], QdMnorm2, appo2, SoS_out;
+  const su2double *P = nullptr;
+
+  bool implicit           = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
+  su2double Gas_Constant  = config->GetGas_ConstantND();
+  bool tkeNeeded          = (config->GetKind_Turb_Model() == SST) || (config->GetKind_Turb_Model() == SST_SUST);
+  bool ratio              = (config->GetActDisk_Jump() == RATIO);
+
+  su2double *Normal = new su2double[MAXNDIM];
```

Comment:
Actually I meant `su2double Normal[MAXNDIM]` like most of your other variables, it's a minor detail.

Reply:
Did it.